src/main/resources/webassets/rest.js

```diff
@@ -122,6 +125,20 @@ base.rest = (function() {
         .then(response => response.json())
         .then(u => objOrError(u, User));
     },
+    addRating: function(target_id, rating) {
+      return baseFetch('/rest/user/' + target_id + '/addrating/' + rating, {
+        method: 'POST',
+        body: JSON.stringify(credentials),
+        headers: jsonHeader,
+      }).then(response => response.json());
+    },
+    setRating: function(id, rating) {
+      return baseFetch('/rest/user/' + target_id + '/setrating/' + rating, {
```

Comment:
`target_id` is not defined here. Should it be `id`?
```suggestion
      return baseFetch('/rest/user/' + id + '/setrating/' + rating, {
```